modules/social_features/social_group/social_group.module

```diff
@@ -229,10 +229,10 @@ function social_group_preprocess_group(array &$variables) {
   $variables['group_type_id'] = $group_type_id;
   $variables['group_type'] = $group->getGroupType()->label();
 
-  // From 10.0.0 onwards, we want to show group types differently.
-  // It will be the value chosen while adding the group instead of saying
-  // 'Flexible group'.
-  if ($group_type_id === 'flexible_group'
+  $group_types = ['flexible_group'];
+  \Drupal::moduleHandler()->alter('social_group_request', $group_types);
+
+  if (in_array($group_type_id, $group_types)
```

Comment:
@rollki @ribel Was there any requirement where groups implementing this alter hook wanted to participate in the "Group type" feature. Please educate me if I am missing some background.